pkg/controller/disruption/disruption.go

```diff
@@ -21,9 +21,8 @@ import (
 	"fmt"
 	"time"
 
-	apps "k8s.io/api/apps/v1beta1"
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/api/extensions/v1beta1"
+	apps "k8s.io/api/apps/v1"
```

Comment:
appsv1

Reply:
Thank you for the advise. and I apologize the late reply.
Other pkg/controller files, deployment and replicaset and so on, have `apps "k8s.io/api/apps/v1"`
Therefore, I changed that way. Should we change to `appsv1` in disruption controller?